### PR TITLE
Chore/rename project

### PR DESCRIPTION
**Pull request by christophe-riolo**
I updated the packaging and renamed the project.
I also had to install `typing-extensions` because we support starting 3.6 but `Literal` was introduced in 3.8

### CODE REVIEW

**Review by ollipa — APPROVED** [review]
Looks good :+1:

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
-name = "flexmock"
-version = "0.10.4"
+name = "flexmock-plus"
+version = "0.10.7"
 description = "flexmock is a testing library for Python that makes it easy to create mocks, stubs and fakes."
```
Comment (adarshk7) [inline pyproject.toml:4]: How about changing the description too?

**Review by adarshk7 — APPROVED** [review]
Looks good!

File: pyproject.toml (hunk 1 above)
Comment (christophe-riolo) [reply pyproject.toml:4]: Do you have suggestions ?

Comment (adarshk7) [reply pyproject.toml:4]: flexmock-plus? 😄

Comment (christophe-riolo) [reply pyproject.toml:4]: Aaah but the library is still flexmock; and it's just temporary anyway, I'll file for the  ownership transfer of an abandoned project in PyPI :)

**Pull request merged**